lib/modules/apostrophe-login/index.js

```diff
@@ -309,12 +310,56 @@ module.exports = {
       return self.apos.users.safe.findOne({
         _id: user._id
       }).then(function(userSafe) {
-        return done(null, userSafe && userSafe.totp && userSafe.totp.key, userSafe && userSafe.totp && userSafe.totp.period);
+        return self.checkIfActive(userSafe, function() {
```

Comment:
You are ignoring any errors from checkIsActive here.

Reply:
Od, done